### PR TITLE
core: Apply static to local functions

### PR DESCRIPTION
**Pull request by lioncash**
Gets rid of missing prototype warnings with clang.

### CODE REVIEW

File: src/core/hle/shared_page.cpp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include "core/core.h"
 #include "core/mem_map.h"
 #include "core/hle/config_mem.h"
+#include "core/hle/shared_page.h"
```
Comment (bunnei) [inline src/core/hle/shared_page.cpp]: This seems unrelated to the commit ^^

Comment (lioncash) [inline src/core/hle/shared_page.cpp]: Nope it's intentional. Without that, the compiler can't see the prototype for Set3DSlider, and thinks it should be declared static.

**Comment by bunnei** [comment]
LGTM
